src/cobalt.rs

```diff
@@ -188,7 +237,11 @@ fn generate_posts(posts: &mut Vec<Document>, context: &Context) -> Result<()> {
             .unwrap_or(liquid::Value::Nil);
         post.attributes.insert("next".to_owned(), next);
 
-        generate_doc(&simple_posts_data, post, context)?;
+        generate_doc(
+            post,
+            context,
+            generate_collections_var(&simple_posts_data, &context),
```

Comment:
Whats the reason for moving the collection stuff to this level?

Reply:
So I can pass either the `collections` object or a `paginator` object

Reply:
Yeah, I though that if we activate pagination, no need to have the posts twice.

Reply:
Makes sense, we just need to make sure we make this clear in the documentation.

Reply:
You might missed it, I've added it to the RFC, under the paginator fields code block.